docs/getting-started.md

```diff
@@ -1,27 +1,17 @@
-# Tekton Pipelines Tutorial
+# Getting Started with Tekton Pipelines
 
-This tutorial uses a simple `Hello World` example to show you how to:
-- Create a `Task`
-- Create a `Pipeline` containing your `Tasks`
-- Use a `TaskRun` to instantiate and execute a `Task` outside of a `Pipeline`
-- Use a `PipelineRun` to instantiate and run a `Pipeline` containing your `Tasks`
+This guide will get you started with tekton pipelines with a simple `Hello World` tutorial.
 
-This tutorial consists of the following sections:
+## Before we begin
 
-- [Creating and running a `Task`](#creating-and-running-a-task)
-- [Creating and running a `Pipeline`](#creating-and-running-a-pipeline)
+Before we begin this tutorial, make sure you have [installed and configured](https://github.com/tektoncd/pipeline/blob/master/docs/install.md)
+the latest release of Tekton on your Kubernetes cluster, including the [Tekton CLI](https://github.com/tektoncd/cli). 
 
-**Note:** Items requiring configuration are marked with the `#configure` annotation.
-This includes Docker registries, log output locations, and other configuration items
-specific to a given cloud computing service.
+If you would like to complete this tutorial on your local workstation, make sure you run clean profile of [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) v1.50 or higher.
 
-## Before you begin
+You will also need an account on [docker hub](https://hub.docker.com).
 
-Before you begin this tutorial, make sure you have [installed and configured](https://github.com/tektoncd/pipeline/blob/master/docs/install.md)
-the latest release of Tekton on your Kubernetes cluster, including the
-[Tekton CLI](https://github.com/tektoncd/cli).
-
-If you would like to complete this tutorial on your local workstation, see [Running this tutorial locally](#running-this-tutorial-locally). To learn more about the Tekton entities involved in this tutorial, see [Further reading](#further-reading).
+Also, make sure you are in the tekton-pipelines namespace if you have tekton installed.
```

Comment:
Don't use "also" repeatedly. A bulleted list is much easier to parse than "Also this," "also that."